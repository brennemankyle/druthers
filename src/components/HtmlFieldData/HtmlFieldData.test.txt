// import React from "react";
// import { shallow } from "enzyme";
// import toJson from "enzyme-to-json";
// import { mockItemList } from "../../mocks";
// import HtmlFieldData from "./HtmlFieldData";

// it("renders", () => {
//   const wrapper = shallow(
//     <HtmlFieldData name="test" itemList={mockItemList} />
//   );

//   expect(toJson(wrapper)).toMatchSnapshot();
// });
