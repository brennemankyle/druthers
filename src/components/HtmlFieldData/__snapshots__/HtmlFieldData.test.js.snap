// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<input
  name="test"
  type="hidden"
  value="1,2,3,4,5"
/>
`;
